Project.toml

```
@@ -1,18 +1,23 @@
 name = "DiffResults"
 uuid = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
-version = "1.1.0"
+version = "1.0.3"
 
 [deps]
-StaticArraysCore = "1e83bf80-4336-4d27-bf5d-d5a4f845583c"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+
+[weakdeps]
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+
+[extensions]
+StaticArraysExt = "StaticArrays"
```

Might be good to use a prefix?
```suggestion
DiffResultsStaticArraysExt = "StaticArrays"
```